core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -504,6 +518,40 @@ private static Map<Integer, ByteBuffer> toReadableByteBufferMap(Map<Integer, Byt
     }
   }
 
+  private static Map<Integer, Long> filteredLongMap(
```

Comment:
we probably can combine these two util method into one with generic, like
```
private static <T> Map<Integer, T> filterColumnsStats(Map<Integer, T> stats, Set<Integer> columnsToKeepStats)
```

Reply:
Done